docker/build-back-publish.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set -uexo pipefail
+
+echo "For local development use only"
+
+REPO=tim-hm
```

Comment:
shouldn’t that rather be on a canonical/ubuntu owned repo?

Reply:
Totally up to you / how you'd want to use this (or not in the future).

I'd suggest it not being the canonical repo because for the 'official' image it should be built by gh workflows and this script would mean locally built images are being mixed into that registry. This script was intended for local testing/development and so in that instance the container registry in use would be the user's gh account.

If that confuses things, it might be better to remove the script altogether?

Reply:
Or maybe build it with the local github registry in that repo? That way, it’s archived here and should be accessible from the same place where the code is hosted.

Reply:
Done!